spec/features/unauth_users/user_visit_menu_spec.rb

```diff
@@ -1,9 +1,14 @@
 require "rails_helper"
 
-RSpec.describe "user visits menu", type: :feature do
-  it "can see all items" do
+RSpec.describe "user visits website", type: :feature do
+  it "can see all items on the menu page" do
     create(:item)
     visit "/menu"
     expect(page).to have_content("Salted Caramel Peanut Butter Cup")
   end
+
+  it "can see store info on the home page" do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.